dynawo/CMakeLists.txt

```diff
@@ -346,7 +337,13 @@ endif()
 find_package(libXML 0.2.4 REQUIRED PATHS ${LIBXML_HOME})
 
 # libIIDM
-find_package(libIIDM 0.2.8 REQUIRED PATHS ${LIBIIDM_HOME}
+if ( CXX11_ENABLED )
+  set( libIIDM_RequiredVersion  "1.2.0")
+  find_package( LibXml2 REQUIRED)
+else ()
+  set( libIIDM_RequiredVersion  "0.2.8")
+endif ()
+find_package(libIIDM "${libIIDM_RequiredVersion}" REQUIRED PATHS ${LIBIIDM_HOME}
   COMPONENTS
     core
     xml
```

Comment:
Those components have no meaning for the new library

Reply:
Changes done in new release